packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -19,6 +19,7 @@ import type {WithActionScheduler} from "@khanacademy/wonder-blocks-timing";
 // NOTE(jeff): Here we share some code for use with PopperJS. Long term,
 // we should either contribute this code to the PopperJS component, or its
 // own non-wonder-blocks package.
+// $FlowIgnore
```

Comment:
I wonder why this wasn't failing before?

Reply:
I introduced this regression in my previous PR (react-window), so it is failing in the current dropdown version. I need to publish this patch and test again in webapp.